modules/rosa-sts-interactive-cluster-creation-mode-options.adoc

```diff
@@ -25,6 +25,9 @@ The following table describes the interactive cluster creation mode options:
 |`OpenShift version`
 |Select the version of OpenShift to install, for example {product-version}. The default is the latest version.
 
+|`Configure the use of IMDSv2 for ec2 instances optional/required (optional)`
```

Comment:
Sorry, the use of the value `optional` vs it's use as a choice was a bit confusing. If setting this is optional, then I suggest rewording this for clarity
```suggestion
|Optional:`Configure the use of IMDSv2 for ec2 instances optional/required`
```

Reply:
Hello @abhatt-rh,
   Thanks for the suggestion. I am going to keep this one as-is to match the others in the list. I understand your take though, thx!